pytorch_lightning/trainer/deprecated_api.py

```diff
@@ -21,28 +22,29 @@ class DeprecatedDistDeviceAttributes:
     _device_type: DeviceType
     _running_stage: RunningStage
     num_gpus: int
+    accelerator_connector: BackendConnector
 
     @property
     def on_cpu(self) -> bool:
         rank_zero_warn("Internal: `on_cpu` is deprecated in v1.2 and will be removed in v1.4.", DeprecationWarning)
-        return self._device_type == DeviceType.CPU
+        return self.accelerator_connector._device_type == DeviceType.CPU
```

Comment:
since you made also the property mapping from `accelerator_connector` lest keep it calling from `self`